docs/infrastructure.md

```diff
@@ -88,15 +96,16 @@ echo \
 apt-get update
 
 apt-get install docker-ce docker-ce-cli containerd.io
-apt-get install python3-pip
-
-adduser --disabled-password --disabled-login orgnode
-usermod -aG docker orgnode
 ```
 
 6. As `orgnode`, set up radicle-client-services:
 
 ```bash
+apt-get install python3-pip
```

Comment:
The heading doesn't match these commands, they should be executed as root.

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/2454/commits/3bac01f09f66792dbce4dec562879710e271b44d